plugins/services/src/js/constants/FrameworkStatus.ts

```diff
@@ -0,0 +1,25 @@
+import { i18nMark } from "@lingui/react";
+
+import { Status } from "./ServiceStatus";
+import StatusIcon from "#SRC/js/constants/StatusIcon";
+
+// prettier-ignore
+const StatusMap: Record<number, Status> = {
```

Comment:
Some of the icons we show for framework statuses are inconsistent what we'd show for service statuses:
"Degraded (Awaiting Resources)" shows `StatusIcon.LOADING`
"Deploying (Awaiting Resources)" shows `StatusIcon.LOADING`
"Degraded (Recovering)" shows `StatusIcon.LOADING`
"Degraded" shows `StatusIcon.LOADING`
"Error Creating Service" shows `StatusIcon.STOPPED`
"Service Unavailable" shows `StatusIcon.ERROR`

Is this intentional?

Reply:
i don't really understand what you mean.

here it says
```
  203: Status(4, i18nMark("Degraded (Awaiting Resources)"), StatusIcon.ERROR),    // not LOADING
  204: Status(2, i18nMark("Deploying (Awaiting Resources)"), StatusIcon.WARNING), // not LOADING
  205: Status(4, i18nMark("Degraded (Recovering)"), StatusIcon.WARNING),          // not LOADING
  206: Status(3, i18nMark("Degraded"), StatusIcon.WARNING),                       // not LOADING
```

also there's no direct match to the service statuses (that i see). 🤔 

shall we sync tomorrow or do you have an idea how to make me understand what you mean?

Reply:
Sure, let's chat tomorrow